lib/en/managing-markers.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: base
+category: 17_Managing Your Applications
+breadcrumb: Managing Your Applications
+parent_url: managing-overview.html
+nav_title: Markers
+nav_priority: 82
+meta_desc: Control your app using markers.
+---
+= Markers
+
+[[top]]
+[float]
+= Markers
+[.lead]
+Marker files are used to set conditions for various stages of a cartridge's life cycle, enabling you to control certain aspects of the cartridge.
```

Comment:
I would reword this, maybe use the version from the [nodejs markers](https://devcenter-jfiala.rhcloud.com/en/node-js-project-structure.html#markers) page

> Marker files can be used to customize various aspects of the build and deployment process.